README.md

```diff
@@ -1,5 +1,103 @@
 # polymer-build
-A library of Gulp build tasks
 
-Note: This is an entirely different library from versions < 0.1.0 by wballard.
-The previous project is available at GitHub here: https://github.com/Custom-Elements/polymer-tools
+Polymer-build is a library for building Polymer projects. While the [Polymer CLI](https://github.com/Polymer/polymer-cli) is still the easiest way to build a project, it's customization options are limited. Polymer-build allows you to create a completely customized build pipeline, which can include additional streams and build tasks in any order. Use polymer-build over the CLI if you:
+
+- Need to customize your build(s) without using command-line flags/arguments
+- Need to run your source code through custom optimizers/processors before, after, or during build
+- Need to hook additional logic into any part of the build process
+
+
+## Installation
+
+```
+npm install --save-dev polymer-build
+```
+
+
+## Usage
+
+ The library was built for Gulp, but it can be used anywhere that Node streams can.
+
+ See the [examples/](examples/) directory for examples of how polymer-build can be used to create a full build-pipeline for your application.
+
+
+### let project = new PolymerProject(options)
+
+PolymerProject gives you access to a collection of streams and helpers for creating your custom build pipeline.
+
+The following options are supported:
+
+```js
+// See ProjectOptions in src/polymer-project.js for more information
+let project = new PolymerProject({
+  root: process.cwd(),
+  entrypoint: 'index.html',
+  shel': 'src/my-app.html',
+  fragments: [
+    'src/my-view1.html',
+    'src/my-view2.html',
+    'src/my-view3.html'
+  ],
+  sourceGlobs: [
+    'src/**',
+    'images/**',
+    'bower.json',
+  ],
+  includeDependencies: [
+  	'bower_components/webcomponentsjs/webcomponents-lite.min.js',
+  ]
+}
+```
+
+### project.sources()
```

Comment:
Add one more level of heading, and replace `polymer.` with so that they appear as sub-headings of `### Polymer Project`